<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillPackEditor</class>
 <widget class="QMainWindow" name="SkillPackEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="SkillsLabel">
        <property name="text">
         <string>Skills:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="skillsList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="AddSkill">
          <property name="text">
           <string>Add skill</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="RemoveSkill">
          <property name="text">
           <string>Remove skill</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="LevelsLabel">
        <property name="text">
         <string>Levels:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="levelsList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="AddLevel">
          <property name="text">
           <string>Add level</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="RemoveLevel">
          <property name="text">
           <string>Remove level</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="LevelDescriptionLabel">
        <property name="text">
         <string>Level description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="descrEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>skillsList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>levelsList</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
